common/sai.py

```diff
@@ -175,8 +175,16 @@ def switch_oid(self, value):
         self.create_alias('SWITCH_ID', 'SAI_OBJECT_TYPE_SWITCH', value)
         self._switch_oid = value
 
-    def process_commands(self, commands):
-        yield from map(self.command_processor.process_command, commands)
+    def process_commands(self, commands, cleanup=False):
+        _commands = []
+        if cleanup:
+            for command in reversed(commands):
+                if command['op'] in ['create', 'remove']:
+                    _commands.append(command.copy())
```

Comment:
How about making the remove commands more compact by only including the name (or key) and op? Stripping the attributes could result in huge memory savings if the list is long (100K entries...)

Reply:
Ok, i agree. I will change this. Thanks, Chris!